### PR TITLE
[KOGITO-1864] Adjust process-kafka-quickstart-springboot example for …

### PR DESCRIPTION
**Pull request by sutaakar**
…Kogito operator

Topics are created automatically.
Tested on OCP 4.3 and locally.

Signed-off-by: Karel Suta <ksuta@redhat.com>

### CODE REVIEW

File: process-kafka-quickstart-springboot/README.md [hunk 1]
```diff
@@ -73,18 +73,6 @@ This quickstart requires an Apache Kafka to be available and by default expects
 
 https://kafka.apache.org/quickstart
 
-In addition to that two topics are needed
```
Comment (radtriste) [inline process-kafka-quickstart-springboot/README.md:76]: this part is no more accurate ?

Comment (sutaakar) [reply process-kafka-quickstart-springboot/README.md:76]: No, topics are created automatically, user doesn't have to do anything.

**Comment by jstastny-cz** [comment]
@sutaakar the example does not use Spring Boot Addon[1] we have, is that expected? 

[1] https://github.com/kiegroup/kogito-runtimes/tree/master/addons/events/kogito-events-spring-boot-addon

**Comment by sutaakar** [comment]
@jstastny-cz That addon is not required as it collects process and task related events, they are usually processed by Data index.
Here we have just a simple example without any external integration with other components. IMHO the addon would have sense in case integration with Data index is implemented.

**Comment by jstastny-cz** [comment]
@sutaakar well, if we deploy the example in OpenShift together with data-index and management-console. The processes won't be visible there now? I thought that was the main goal of the example to show exactly that integration ... otherwise we have IMO just a single example with events and that process-quarkus-example .... except of the travel agency app.

**Comment by sutaakar** [comment]
Well, this process doesn't have any user task or persistence - you just send a JMS message and get different message back.
IMHO there won't be anything to be shown in Data index or Management console as it shows only existing processes and tasks (right?).

**Comment by jstastny-cz** [comment]
Ah, ok, in that case I was wrong about the purpose of the example :-) thanks

**Review by radtriste — APPROVED** [review]

**Review by Sgitario — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]
Well done! Many thanks for this.

**Pull request merged**